markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -134,14 +134,14 @@ library PerpsAccount {
             revert AccountLiquidatable(accountId);
         }
 
+        availableWithdrawableCollateralUsd = availableMargin - requiredMaintenanceMargin;
+
```

Comment:
why is this a bug?  it can be after the revert.  makes no real difference since the middle if statement is a revert.  

essentially its:
- if any of these if statements are true, the call will revert.
- otherwise set the `availableWithdrawableCollateralUsd`.

Reply:
Hmm are you sure, the new withdraw test is failing with out this change?